tests/unpaid-work-and-delius/create-upw-assessment.spec.ts

```diff
@@ -42,8 +46,12 @@ test('Create a UPW-Assessment from Delius and verify the Pdf is uploaded back to
 
     // Then the document appears in the Delius document list
     await page.locator('a', { hasText: 'Document List' }).click()
+    await page.locator('[title="Button to search for Documents"]').click()
+
+    // await page.locator('#documentListForm\\:j_id_id70').click()
     await expect(page.locator('#documentListForm\\:documentDrawerTable\\:tbody_element')).toContainText(
-        'CP/UPW Assessment'
+        'CP/UPW Assessment',
+        { timeout: 15000 }
     )
```

Comment:
I think we should replace this with a doUntil or refreshUntil, as it might take a little while for the message to come across.  I don't think the extra timeout will do anything because it's not refreshing the page.

E.g. something like:
```typescript
    await page.locator('a', { hasText: 'Document List' }).click()

    doUntil(
        () => page.getByRole('button', { name: 'Search' }).click(),
        () => expect(page.locator('table')).toContainText('CP/UPW Assessment')
    )
```